app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -140,20 +177,22 @@ public void testGetPrivate() {
         assertTrue(mood.getPrivate());
     }
 
-    /*
-    public void testSetMapPosition() {
+    public void testSettMapPosition() {
         Mood mood = new Mood();
-        Location loc = null; //TODO: properly initialize a test loc
-        mood.setMapPosition(loc);
-        assertNotNull(mood.getMapPosition());
+        Double lat = 123213.000324;
```

Comment:
Yes, in the actual case where we call they will be derived from our map with the .location() function